### PR TITLE
Adding FireSlime games

### PR DESCRIPTION
**Pull request by luanpotter**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -30,12 +30,24 @@
 
 ### Open Source
 
+#### Endeless Runner
```
Comment (impulse) [inline readme.md]: Should be Endless

File: readme.md [hunk 2]
```diff
@@ -30,12 +30,24 @@
...
+
+ - [**BGUG**](https://github.com/fireslime/bgug) - Break Guns Using Gems is a fast paced side-scrolling platformer with innovative controls and and a gun-related twist. By [FireSlime](https://fireslime.xyz)
+
```
Comment (impulse) [inline readme.md]: add `- [Add Your Item](https://github.com/flame-engine/awesome-flame/pulls)`

File: readme.md [hunk 3]
```diff
@@ -30,12 +30,24 @@
...
+#### Casual
+
+ - [**Bob Box**](https://github.com/fireslime/bounce_box) - Bob Box is a simple casual game writen in Flutter+Flame. By [FireSlime](https://fireslime.xyz)
+
```
Comment (impulse) [inline readme.md]: add `- [Add Your Item](https://github.com/flame-engine/awesome-flame/pulls)`

File: readme.md [hunk 4]
```diff
@@ -30,12 +30,24 @@
...
 #### Other Genre?
 
 - [Add Your Item](https://github.com/flame-engine/awesome-flame/pulls)
 
 ### App Releases
 
+#### Metroidvania
+
+ - [**Tales of a Lost Mine**](https://fireslime.xyz/games/tales.html) - [Android](https://play.google.com/store/apps/details?id=xyz.fireslime.tales&pcampaignid=MKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1) - Tales of a Lost Mine is an awesome platform-adventure Metroidvania. Get in the role of a brave explorer and uncover the secrets of a long lost time. By [FireSlime](https://fireslime.xyz)
+
```
Comment (impulse) [inline readme.md]: add `- [Add Your Item](https://github.com/flame-engine/awesome-flame/pulls)`

File: readme.md [hunk 5]
```diff
@@ -30,12 +30,24 @@
 
 ### Open Source
 
+#### Endeless Runner
+
+ - [**BGUG**](https://github.com/fireslime/bgug) - Break Guns Using Gems is a fast paced side-scrolling platformer with innovative controls and and a gun-related twist. By [FireSlime](https://fireslime.xyz)
+
+#### Casual
```
Comment (impulse) [inline readme.md]: Genres should be ordered A-Z

File: readme.md [hunk 6]
```diff
@@ -30,12 +30,27 @@
 
 ### Open Source
 
+#### Casual
+
+ - [**Bob Box**](https://github.com/fireslime/bounce_box) - Bob Box is a simple casual game writen in Flutter+Flame. By [FireSlime](https://fireslime.xyz)
```
Comment (impulse) [inline readme.md]: Description feels a bit repetitive

Suggestion:
`- [**Bob Box**](https://github.com/fireslime/bounce_box) - Asimple casual game by [FireSlime](https://fireslime.xyz)`

File: readme.md [hunk 7]
```diff
@@ -30,12 +30,27 @@
...
+ - [Add Your Item](https://github.com/flame-engine/awesome-flame/pulls)
+
+#### Endless Runner
+
+ - [**BGUG**](https://github.com/fireslime/bgug) - Break Guns Using Gems is a fast paced side-scrolling platformer with innovative controls and and a gun-related twist. By [FireSlime](https://fireslime.xyz)
```
Comment (impulse) [inline readme.md]: We need to have a common standard. I am okay with using `. By X` in favor of `by X` but then we need to adjust contributing.md

File: readme.md [hunk 8]
```diff
@@ -30,12 +30,27 @@
...
+
 #### Other Genre?
 
 - [Add Your Item](https://github.com/flame-engine/awesome-flame/pulls)
 
 ### App Releases
 
+#### Metroidvania
+
+ - [**Tales of a Lost Mine**](https://fireslime.xyz/games/tales.html) - [Android](https://play.google.com/store/apps/details?id=xyz.fireslime.tales&pcampaignid=MKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1) - Tales of a Lost Mine is an awesome platform-adventure Metroidvania. Get in the role of a brave explorer and uncover the secrets of a long lost time. By [FireSlime](https://fireslime.xyz)
```
Comment (impulse) [inline readme.md]: Description suggestion:
>An awesome platform-adventure Metroidvania. Get in the role of a brave explorer and uncover the secrets of a long lost time

File: readme.md (hunk 7 above)
Comment (luanpotter) [reply readme.md]: I can change to `by X`

Comment (luanpotter) [reply readme.md]: actually I think `. By X` or `, by X` are better, what do you think?

Comment (impulse) [reply readme.md]: Let us do `. By X` then :)

**Pull request merged**